scalardb-graphql-sample/README.md

```diff
@@ -1,130 +1,222 @@
+### Clone the ScalarDB samples repository
 
-## Set up
+Open **Terminal**, then clone the ScalarDB samples repository by running the following command:
 
-This section describes the steps to start the database and the GraphQL server and build the client application.
+```console
+$ git clone https://github.com/scalar-labs/scalardb-samples
+```
+
+Then, go to the directory that contains the sample application by running the following command:
+
+```console
+$ cd scalardb-samples/scalardb-graphql-sample
+```
 
 ### Start Cassandra
 
-First, let's start the Cassandra server with `docker-compose`:
+To start the Cassandra server, which is included in the Docker container for the sample application, make sure Docker is running and then run the following command:
+
+```console
+$ docker-compose up -d cassandra
+```
+
+{% capture notice--info %}
+**Note**
+
+Starting the Docker container may take more than one minute depending on your development environment.
+
+To check the logs, run the following command:
 
 ```console
-docker-compose up -d cassandra
+$ docker-compose logs -f
 ```
+{% endcapture %}
 
-You need to wait for the Cassandra to start up. Checking the logs with the `docker-compose logs -f` command would be helpful.
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-### Load schema
+### Load the schema
 
-In this step, you will load the database schema for the sample app (`emoney-schema.json`) to the Cassandra database.
-To download the schema loader tool, `scalardb-schema-loader-<VERSION>.jar`, see the [Releases](https://github.com/scalar-labs/scalardb/releases) of ScalarDB and download the version that you want to use.
-You can load the schema to the database by the following command:
+The database schema (the method in which the data will be organized) for the sample application has already been defined in [`emoney-schema.json`](emoney-schema.json).
 
-```shell
+To apply the schema, go to the [ScalarDB Releases](https://github.com/scalar-labs/scalardb/releases) page and download the ScalarDB Schema Loader that matches the version of ScalarDB that you want to use to the `scalardb-samples/scalardb-graphql-sample` folder.
+
+Then, run the following command, replacing `<VERSION>` with the version of the ScalarDB Schema Loader that you downloaded:
+
+```console
 $ java -jar scalardb-schema-loader-<VERSION>.jar --config database.properties --schema-file emoney-schema.json --coordinator
 ```
 
-### Start GraphQL server
+### Start the GraphQL server
+
+Before starting the GraphQL server, log in to the GitHub Container registry by running the following command and entering your GitHub credentials as instructed:
+
+```console
+$ docker login ghcr.io
+```
```

Comment:
I added this part since I needed to run this command in order to run the GraphQL server. Do we need to add any additional information here?

Reply:
@brfrn169 Could you please help me with this part? Is this new part sufficient?🙇🏻‍♂️